pallets/orml-currencies-allowance-extension/src/default_weights.rs

```diff
@@ -75,10 +90,61 @@ impl<T: frame_system::Config> crate::WeightInfo for WeightInfo<T> {
 		// Proof Size summary in bytes:
 		//  Measured:  `561`
 		//  Estimated: `14248`
-		// Minimum execution time: 31_000_000 picoseconds.
-		Weight::from_parts(33_000_000, 0)
-			.saturating_add(Weight::from_parts(0, 14248))
-			.saturating_add(T::DbWeight::get().reads(4))
-			.saturating_add(T::DbWeight::get().writes(3))
+		// Minimum execution time: 43_000_000 picoseconds.
+		Weight::from_parts(44_000_000, 14248)
+			.saturating_add(T::DbWeight::get().reads(4_u64))
+			.saturating_add(T::DbWeight::get().writes(3_u64))
 	}
 }
+
+// For backwards compatibility and tests
+impl WeightInfo for () {
+	/// Storage: TokenAllowance AllowedCurrencies (r:0 w:1)
+	/// Proof Skipped: TokenAllowance AllowedCurrencies (max_values: None, max_size: None, mode: Measured)
+	fn add_allowed_currencies() -> Weight {
```

Comment:
At the begining, in order for the project to build, I needed to replace this function signature with this one:
```rust
fn add_allowed_currencies(_r: u32) -> Weight
```